EFCache/ICache.cs

```diff
@@ -41,5 +41,16 @@ public interface ICache
         /// </summary>
         /// <param name="key">The cache key.</param>
         void InvalidateItem(string key);
+
+        /// <summary>
+        /// Calculates and returns the cache size in bytes
+        /// </summary>
+        /// <returns>The cache size in bytes</returns>
+        int GetCacheSize();

```

Comment:
Why would you need this method if the cache is for instance backed by Redis? How would you implement this?